### ISSUE TITLE
svg dom not rendering

### ISSUE THREAD

**Issue by colombod** [microsoft/vscode#112693]
- VSCode Version:1.53.0-insider (user setup)
- OS Version: Windows_NT x64 10.0.19042

Steps to Reproduce:

1. Install [.NET Interactive Notebook extensions](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.dotnet-interactive-vscode)
2. Open the notebook in the attached zip file using the .NET Interactive command from the palette
3. Run all notebook and this is the output 
![image](https://user-images.githubusercontent.com/375556/102421134-9db29600-3ffb-11eb-809e-6a5597fcfef8.png)
4. Collapse the cell  output
5. Expand cell output and you see this
 
![image](https://user-images.githubusercontent.com/375556/102421176-bd49be80-3ffb-11eb-8af1-997a9b9904c9.png)

Does this issue occur when all extensions are disabled?: Yes
[d3js.zip](https://github.com/microsoft/vscode/files/5706397/d3js.zip)

.NET Interactive version
![image](https://user-images.githubusercontent.com/375556/102425989-62699480-4006-11eb-808f-aea2ebcd9a22.png)

**Comment by colombod** [comment]
@rebornix any idea?

**Comment by rebornix** [comment]
@colombod I'm not getting it to work at all and in Dev Tools console, I saw

```
Uncaught (in promise) TypeError: notebookScope.plot is not a function
```

I wonder if it's because `notebookScope` was not initialized then it gets executed in the webview.

**Comment by colombod** [comment]
I don't get that issue at all

**Comment by colombod** [comment]
![image](https://user-images.githubusercontent.com/375556/102425361-1538f300-4005-11eb-9a5e-c1a9399610a7.png)

**Comment by colombod** [comment]
The element is in the dom but not rendered 
![image](https://user-images.githubusercontent.com/375556/102425515-6648e700-4005-11eb-8d9d-d2fc3ffd2245.png)

**Comment by colombod** [comment]
This file includes the notebook to repro
[d3js.zip](https://github.com/microsoft/vscode/files/5706394/d3js.zip)

**Comment by rebornix** [comment]
bravo, I can reproduce with the file, will investigate

**Comment by rebornix** [comment]
Reproduce steps:

* Install https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.dotnet-interactive-vscode
* download https://github.com/microsoft/vscode/issues/112693#issuecomment-747142338, unzip and open the file in VS Code Insiders
* Run cells from top down
* With electron 9, the svg shows up after 2 seconds
* With electron 11, the svg was rendered but doesn't display. However, as long as there is any CSS or layout change in the webview/iframe, svg will display right after (for example, uncheck any css rule applied to `body`.)

**Comment by rebornix** [comment]
Another interesting finding:

* After running all the cells, the svg doesn't display, and then you switch to another application and switch back to VS Code, the svg comes back

**Comment by deepak1556** [comment]
I wasn't able to make progress in narrowing down the issue, @rebornix I need your help with creating a minimal repro outside notebook, maybe we can do this in debt week ?

**Comment by rebornix** [comment]
@deepak1556 sounds good to me.

**Comment by colombod** [comment]
@rebornix , @deepak1556 let me know if I can help

**Comment by rzhao271** [comment]
Turns out this issue doesn't have to do with webviews or iframes, but has to do with an SVG rendering glitch in Chromium where invalidate wasn't getting called in some cases, including this one: https://chromium.googlesource.com/chromium/src/+/15a8dbe6efe865eaaae63b48b6a3bacd3420a89b.
This fix is in Electron starting in v12.0.0-beta.16.
Hoping to backport the fix later to 11-x-y, but at least we got the root cause down.
